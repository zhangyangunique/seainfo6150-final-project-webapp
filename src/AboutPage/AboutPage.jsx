import React from "react";
import styles from "./AboutPage.module.css";

const AboutPage = () => {
	return (
		<div className={styles.aboutContainer}>
			<img
				className={styles.aboutImage}
				src="https://images.unsplash.com/photo-1544716278-ca5e3f4abd8c?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1334&q=80"
				alt="about us"
			/>
			<h1 className={styles.aboutTitle}>About me</h1>
			<div className={styles.information}>
				<p>Name: Yang Zhang</p>
				<address>
					Email:
				<a href="mailto:zhang.yang2@northeastern.edu">&nbsp;zhang.yang2@northeastern.edu</a>
					<br />
				</address>
				<p>College: Northeastern University</p>
				<p>Location: San Jose</p>
			</div>
		</div>
	);
};

export default AboutPage; 