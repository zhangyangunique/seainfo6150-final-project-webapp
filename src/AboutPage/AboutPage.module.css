.aboutContainer {
	font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
	font-size: 30px;
	text-align: center;
}

.aboutImage {
	width: 30%;
	margin: 40px auto;
	display: block;
}

a {
	text-decoration: none;
}

.information {
	text-align: center;
}