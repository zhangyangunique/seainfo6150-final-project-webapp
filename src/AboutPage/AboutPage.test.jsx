import React from "react";
import { render } from "@testing-library/react";
import AboutPage from "./AboutPage";

describe("AboutPage tests", () => {
    it("renders correctly", () => {
        const { container } = render(<AboutPage />);
        expect(container).toMatchSnapshot();
    });
});