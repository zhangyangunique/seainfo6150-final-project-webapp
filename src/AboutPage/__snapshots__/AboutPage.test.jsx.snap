// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage tests renders correctly 1`] = `
<div>
  <div
    class="aboutContainer"
  >
    <img
      alt="about us"
      class="aboutImage"
      src="https://images.unsplash.com/photo-1544716278-ca5e3f4abd8c?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1334&q=80"
    />
    <h1
      class="aboutTitle"
    >
      About me
    </h1>
    <div
      class="information"
    >
      <p>
        Name: Yang Zhang
      </p>
      <address>
        Email:
        <a
          href="mailto:zhang.yang2@northeastern.edu"
        >
          Â zhang.yang2@northeastern.edu
        </a>
        <br />
      </address>
      <p>
        College: Northeastern University
      </p>
      <p>
        Location: San Jose
      </p>
    </div>
  </div>
</div>
`;
